lib/process-data-availability.js

```diff
@@ -120,9 +128,7 @@ const isIdxField = (resourceName, fieldName, standardFieldCache = {}) =>
   resourceName &&
   fieldName &&
   isResoField(resourceName, fieldName, standardFieldCache) &&
-  !!standardFieldCache[resourceName][fieldName]?.payloads?.filter(
-    (x) => x === 'IDX'
-  ).length > 0;
+  !!standardFieldCache[resourceName][fieldName]?.payloads?.some(x => x === 'IDX');
```

Comment:
Could probably use optional chaining here with `standardFieldCache[resourceName]?.[fieldName]...` but we can address this in another set of changes. Since the isResoField check is being done first, it should guard against any issues here.